docs/book/src/user/quick-start.md

```diff
@@ -769,582 +148,22 @@ kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o json \
   > ./capi-quickstart.kubeconfig
 ```
 
-When using docker-for-mac MacOS, you will need to do a couple of additional
-steps to get the correct kubeconfig:
-
-```bash
-# Point the kubeconfig to the exposed port of the load balancer, rather than the inaccessible container IP.
-sed -i -e "s/server:.*/server: https:\/\/$(docker port capi-quickstart-lb 6443/tcp | sed "s/0.0.0.0/127.0.0.1/")/g" ./capi-quickstart.kubeconfig
+### Deploy a CNI solution
 
-# Ignore the CA, because it is not signed for 127.0.0.1
-sed -i -e "s/certificate-authority-data:.*/insecure-skip-tls-verify: true/g" ./capi-quickstart.kubeconfig
-```
-
-{{#/tab }}
-{{#/tabs }}
-
-Deploy a CNI solution, Calico is used here as an example.
-
-{{#tabs name:"tab-usage-addons" tabs:"Calico"}}
-{{#tab Calico}}
+Calico is used here as an example.
 
 ```bash
 kubectl --kubeconfig=./capi-quickstart.kubeconfig \
   apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
 ```
```

Comment:
This guide kind of leaves things a bit abruptly. Should we link to the `clusterctl` docs so that users can discover other actions?

I suspect we should also lead users to a more advanced page for workload clusters as well. For example we should at least let them know about scaling up/down the KubeadmControlPlane and MachineDeployment resources.

Reply:
I have linked the clusterctl docs which are already there.
WRT to scaling up/down the KubeadmControlPlane and MachineDeployment resources, I'm 100% ok with the idea, but this is dependent by how the cluster-template is compiled, so there is the risk it won't work for all the providers.